### ISSUE TITLE
Push Notifications work on Simulator for Andriod and IOS but not on actual devices

### ISSUE THREAD

**Issue by Rob97** [MicrosoftDocs/azure-dev-docs#965]
After following all of the steps in the documentation, I get a 401 unauthorized when trying to register from the device but it works perfectly on simulators. 

So I don't understand how can it be an unauthorized issue when the code isn't changing, but only the device is. 
---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: ae6b97ae-ec64-6de9-8f02-16f91f5f9519
* Version Independent ID: d9748b97-4afb-fc97-9598-188bd460cc13
* Content: [Send push notifications to Xamarin.Forms apps using Azure Notification Hubs via a backend service](https://learn.microsoft.com/en-us/azure/developer/mobile-apps/notification-hubs-backend-service-xamarin-forms#creating-a-certificate-for-notification-hubs)
* Content Source: [articles/mobile-apps/notification-hubs-backend-service-xamarin-forms.md](https://github.com/MicrosoftDocs/azure-dev-docs/blob/main/articles/mobile-apps/notification-hubs-backend-service-xamarin-forms.md)
* Service: **mobile-services**
* GitHub Login: @mikeparker104
* Microsoft Alias: **miparker**

**Comment by Rob97** [comment]
If this issue persists for anyone, I recommend checking the provisioning profile being used.

**Comment by Rob97** [comment]
When I submit to test flight, the entire system collapsed. It still works on andriod simluator, but the iphone simulator and device stopped functioning completely even though no code was changed. 

It works extremely randomly. I have trippled checked the provisioning profiles and certificates, deleting and recreating them multiple times to ensure there is no mix up. I did remove communication notifications form the entitlements, is this why it is failing?
